bodhi/server/models.py

```diff
@@ -1756,13 +1764,15 @@ def set_request(self, db, action, username):
                         'stable repository.  It must first reach a karma of %s, consisting of %s '
                         'positive karma from proventesters, along with %d additional karma from '
                         'the community. Or, it must spend %s days in testing without any negative '
-                        'feedback')
+                        'feedback.')
```

Comment:
I'm +1 to fixing the grammar here, but please do it in a separate pull request since it is not related to adding Greenwave functionality to Bodhi.